src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,37 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gapDay;
+  width: $widthCalendar;
+
+  @each $number, $day in $weekDays {
+    &--start-day--#{$day} :first-child {
+      margin-left: ($number - 1) * ($sizeDay + $gapDay);
+    }
+  }
+
```

Comment:
hmm, i can't have a 31-day month? i should be able to.
neither can i show 28 days
`class="calendar calendar--start-day--tue calendar--month-length--28"`
you need another `@for` loop for this 

consult with Ashkar brothers if you don't know how to approach it, k?